source/Plugins.h

```diff
@@ -67,13 +68,32 @@ struct Plugin {
 	bool enabled = true;
 	// The current state of the plugin.
 	bool currentState = true;
+	// If this plugin has been deleted.
+	bool removed = false;
 };
 
 
 
 // Tracks enabled and disabled plugins for loading plugin data or skipping it.
 // This object is updated by toggling plugins in the Preferences UI.
 class Plugins {
+public:
+
+	struct InstallData
+	{
+		std::string name;
+		std::string url;
+		std::string version;
+		std::string aboutText;
+		bool installed = false;
+		bool outdated = false;
+		InstallData(std::string name = "", std::string url = "", std::string version = "",
+			std::string aboutText = "", bool installed = false, bool outdated = false)
+		: name(name), url(url), version(version), aboutText(aboutText), installed(installed),
+			outdated(outdated) {}
```

Comment:
Why is it defined in the header?

Reply:
dunno seems to be standard, see projectile::impactinfo for example